schema/workbook.json

```diff
@@ -1,2 +1,754 @@
+        },
+        "gridSettings": {
+            "$id": "#/definitions/gridSettings",
+            "type": "object",
+            "properties": {
+                "formatters": {
+                    "type": "array",
+                    "description": "the formatters applied to the columns",
+                    "items": [
+                        {
+                            "$ref": "#/definitions/formatters"
+                        }
+                    ]
+                },
+                "labelSettings": {
+                    "type": "array",
+                    "items": [
+                        {
+                            "$ref": "#/definitions/gridLabelSettings"
+                        }
+                    ],
+                    "description": "Settings for custom column labels"
+                },
+                "filter": {
+                    "type": "boolean",
+                    "description": "Show the filter search box above the grid"
+                },
+                "rowLimit": {
+                    "type": "integer",
+                    "description": "The max number of rows to display for this grid"
+                },
+                "hierarchySettings": {
+                    "type": "object",
+                    "properties": {
+                        "treeType": {
+                            "type": "integer",
+                            "description": "The hierarchy type"
+                        },
+                        "groupBy": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            },
+                            "description": "The column ids to group by"
+                        },
+                        "expandTopLevel": {
+                            "type": "boolean",
+                            "description": "Expand the top level of the grid by default"
+                        },
+                        "finalBy": {
+                            "type": "string",
+                            "description": "The column id to finally group by"
+                        }
+                    }
+                },
+                "sortBy": {
+                    "type": "array",
+                    "items": [
+                        {
+                            "$ref": "#/definitions/sortBy"
+                        }
+                    ]
+                }
+            }
+        },
+        "gridLabelSettings": {
+            "$id": "#/definitions/gridLabelSettings",
+            "type": "object",
+            "properties": {
+                "columnId": {
+                    "type": "string",
+                    "description": "The id of the column"
+                },
+                "label": {
+                    "type": "string",
+                    "description": "The label of the column header",
+                    "tags": [
```

Comment:
here, do we haev any way of getting the comments to the localizers?  we tag these fields as localizable, but the comments here and in series labels aren't marked in any way so that the localizers see them?

Reply:
From the email thread it doesn't seem like it, but we could ask